hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastConfigBeanDefinitionParser.java

```diff
@@ -682,6 +687,12 @@ public void handleRingbufferStoreConfig(Node node, BeanDefinitionBuilder ringbuf
             ringbufferConfigBuilder.addPropertyValue("ringbufferStoreConfig", builder.getBeanDefinition());
         }
 
+        public void handleAtomicLong(Node node) {
+            final BeanDefinitionBuilder atomicLongConfigBuilder = createBeanBuilder(AtomicLongConfig.class);
```

Comment:
Have you become a final person? :)

Reply:
It's the style of the whole class, didn't want to change it in this PR.

Reply:
Yeeeey!